### PR TITLE
DEVELOPER-3647 - Added html comments to clear cache

### PR DESCRIPTION
**Pull request by Dantheman720**

### CODE REVIEW

**Comment by Dantheman720** [comment]
@LightGuard - Do you think adding the html comment will work?

**Comment by LightGuard** [comment]
Should work fine, are you seeing the change in the PR site?

**Comment by Dantheman720** [comment]
@LightGuard - Looks like there are also issues with Youtube videos on cdk. I'll add the https change to that as well.

**Comment by Dantheman720** [comment]
retest this please

File: Gemfile [hunk 1]
```diff
@@ -53,7 +53,7 @@ gem 'listen', '3.0.8'
 #
 # From a location on your disk:
 #
-gem 'aweplug', git: 'https://github.com/awestruct/aweplug'
+gem 'aweplug', git: 'https://github.com/Dantheman720/aweplug', :branch => 'DEVELOPER-3647'
```
Comment (LightGuard) [inline Gemfile]: This needs to be sent to aweplug and merged in. There's no PR yet.

Comment (Dantheman720) [inline Gemfile]: I submitted the PR. I'll resubmit this PR once Aweplug is merged. This was mainly for testing purposes.

**Comment by Dantheman720** [comment]
@LightGuard @KamiQuasi @adelasofia - This should be good to merge now.

Example where Vimeo was the issue: https://developers-pr.stage.redhat.com/pr/1609/export/products/datagrid/overview/

Example where Youtube was the issue: https://developers-pr.stage.redhat.com/pr/1609/export/products/cdk/overview/

**Pull request merged**